.github/workflows/release.yml

```diff
@@ -7,22 +7,25 @@ jobs:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Prepare repository
         run: git fetch --unshallow --tags
 
       - name: Use Node.js 20.x
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 20.x
 
       - name: Install dependencies
-        uses: bahmutov/npm-install@v1
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+          run_install: true
 
       - name: Create Release
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: |
-          yarn release
+          npm run release
```

Comment:
shouldn't all these `npm run` be `pnpm run` instead to use the correct package resolution, or am I misunderstanding something? Same for the lines in `package.json`?

Reply:
Yeah, that was silly of me. Done in [4269195](https://github.com/storybookjs/addon-svelte-csf/pull/113/commits/4269195b43956bc2f7401f480a9012c9ed660581).